File: src/ru/guychokalolo/transport/MotorCycle.java
package ru.guychokalolo.transport;

public class MotorCycle extends Transport {

    public MotorCycle(String color, int wheel, int numberOfPersons, String name) {
        super(color, wheel, numberOfPersons, name);
    }

    @Override
    public String getInfoTransport() {
        return "color = " + color + ", " + "wheel = " + Integer.toString(wheel)+
                ", " + "numberOfPersons = " + Integer.toString(numberOfPersons) + ", " + "name = " +name ;
    }
}

File: src/ru/guychokalolo/kotlin/kotlin/transport/MotorCycle.kt
package transport

class MotorCycle(name: String,
                 color: String,
                 wheel: Int,
                 numberOfPersons: Int) : Transport(name, color, wheel,numberOfPersons) {


    override fun getInfoTransport() : String {
        return "$name, $color, $wheel, $numberOfPersons"
    }
}